core/src/test/java/org/apache/calcite/test/JdbcTest.java

```diff
@@ -2385,17 +2385,17 @@ private void checkNullableTimestamp(CalciteAssert.Config config) {
   @Test public void testReuseExpressionWhenNullChecking3() {
     CalciteAssert.hr()
         .query(
-            "select substring(\"name\", \"deptno\"+case when user <> 'sa' then 1 end) from \"hr\".\"emps\"")
+            "select substring(\"name\", \"deptno\"+case when CURRENT_PATH <> '' then 1 end) from \"hr\".\"emps\"")
```

Comment:
Is it possible to set `user` to `sa` for current test instead of changing the query ?